phpunit-dama-doctrine.xml.dist

```diff
@@ -13,7 +13,7 @@
         <env name="KERNEL_CLASS" value="Zenstruck\Foundry\Tests\Fixtures\Kernel" />
         <env name="USE_DAMA_DOCTRINE_TEST_BUNDLE" value="1" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0&amp;max[direct]=0"/>
-        <env name="SHELL_VERBOSITY" value="-1"/>
+        <env name="SHELL_VERBOSITY" value="0"/>
```

Comment:
I had hard time making the test suite work with mongo + php on docker because a SSL library was missing in php's Dockerfile.

Problem is with the error was not displaying anywhere. Here was the output of the error:
```shell
$ make phpunit-mongo
PHPUnit 8.5.30 #StandWithUkraine

Testing Project Test Suite
SSSSSSSSSSSSSSSSSSSSE

Time: 507 ms, Memory: 32.00 MB

There was 1 error:

1) Zenstruck\Foundry\Tests\Functional\FactoryDoctrineCascadeTest::many_to_one_relationship
RuntimeException: Error running "doctrine:mongodb:schema:create": 

/app/src/Test/AbstractSchemaResetter.php:24
/app/src/Test/ODMSchemaResetter.php:42
/app/src/Test/DatabaseResetter.php:53
/app/src/Test/ResetDatabase.php:60
```

This is because of `SHELL_VERBOSITY=-1`: `AbstractSchemaResetter` uses command's output.
With shell verbosity=0 you now can see the error:
```shell
$ make phpunit-mongo
PHPUnit 8.5.30 #StandWithUkraine

Testing Project Test Suite
SSSSSSSSSSSSSSSSSSSSE

Time: 507 ms, Memory: 32.00 MB

There was 1 error:

1) Zenstruck\Foundry\Tests\Functional\FactoryDoctrineCascadeTest::many_to_one_relationship
RuntimeException: Error running "doctrine:mongodb:schema:create": The SCRAM_SHA_256 authentication mechanism requires libmongoc built with ENABLE_SSL

/app/src/Test/AbstractSchemaResetter.php:24
/app/src/Test/ODMSchemaResetter.php:42
/app/src/Test/DatabaseResetter.php:53
/app/src/Test/ResetDatabase.php:60
```

Reply:
yes :tada: 

the trick was to add some ssl libraries in the Dockerfile. But I recommend to keep the `SHELL_VERBOSITY=0`, since it does not pollute the tests with extra output, but we can see potential errors of doctrine commands under the hood